cmd/cadvisor.go

```diff
@@ -87,6 +87,7 @@ var (
 		container.ProcessSchedulerMetrics:        struct{}{},
 		container.ProcessMetrics:                 struct{}{},
 		container.HugetlbUsageMetrics:            struct{}{},
+		container.ReferencedMetric:               struct{}{},
```

Comment:
nit: even though we only have one metric, lets still use the plural: `ReferencedMetrics`.